### DISCUSSION TITLE
Is SQLBoiler (talking to Postgres) escaping my text fields with HTML entities?

### DISCUSSION THREAD

**Discussion by glenatron** [aarondl/sqlboiler#1358] [Q&A]
I'm using SQLBoiler with Postgres for the back-end of a project and one thing I have noticed happening is that when I save text fields that contain single or double quotes they're being quoted using the html `&#39;` and `&#34;` entity codes. That isn't a big problem except that they're not unescaped on the way back out so I'm currently having to do that manually. 

As far as I can tell the standard postgres libraries escape quotes using a backslash rather than entity codes - is this something SQL Boiler does, and if so is there a list of everything it will escape that way or a way to automatically unescape the same values on the way back out?

**Comment by stephenafamo** [comment]
This is not SQLBoiler, instead, I believe it is just how those entities are escaped when submitted through a form. You can print out the form value to confirm.

**Reply by glenatron** [reply]
The Go debugger is showing them as regular single and double quotes when they are added as property fields to my SQLBoiler models, when the models are loaded back in from Postgres they are entity-encoded. I'm happy to believe it isn't SQLBoiler, but it has to be happening somewhere between there and the database.
